src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -128,7 +131,13 @@
 
         topics.lookup(CONFIGURATION_CONFIG_KEY, LOGS_UPLOADER_PERIODIC_UPDATE_INTERVAL_SEC)
                 .dflt(DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC)
-                .subscribe((why, newv) -> periodicUpdateIntervalSec = Coerce.toInt(newv));
+                .subscribe((why, newv) -> {
+                    if (why == WhatHappened.removed) {
+                        periodicUpdateIntervalSec = DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC;
+                    } else {
+                        periodicUpdateIntervalSec = Coerce.toInt(newv);
```

Comment:
check if int is > than some minimum - do we want to allow 1s, negative numbers, 0, etc...

Reply:
As discussed offline, I'll check > 0, and if it fails, I'll log a warning and use the default.